spec/services/scrapers/low_priority_spec.rb

```diff
@@ -4,15 +4,17 @@
 
 RSpec.describe Scrapers::LowPriority do
   describe '#perform' do
-    it 'add specs' do
+    it 'adds jobs to the CourtCaseWorker' do
       test_case = create(:court_case, :with_html, closed_on: 7.months.ago)
       test_case.case_html.update(scraped_at: 7.months.ago)
-
-      data = described_class.perform
+      data = nil
 
       expect do
-        described_class.perform
+        data = described_class.perform
       end.to change(CourtCaseWorker.jobs, :size).by(1)
+      expect do
+        described_class.perform
+      end.to change(CourtCaseWorker.jobs, :size).by(0)
```

Comment:
in general we only want one assertion per `it` block (so move this to it's own)
you can then move the setup to a `before` block